includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -924,7 +924,7 @@ function() use ( $payment_fields ) {
 	 */
 	public function get_payment_method_ids_enabled_at_checkout( $order_id = null, $force_currency_check = false ) {
 		$capture                    = empty( $this->get_option( 'manual_capture' ) ) || $this->get_option( 'manual_capture' ) === 'no';
-		$capturable_payment_methods = $capture ? $this->get_upe_enabled_payment_method_ids() : [ 'card' ];
+		$capturable_payment_methods = $capture ? $this->get_upe_enabled_payment_method_ids() : [ 'card', 'link' ];
```

Comment:
We have constants available for the payment methods see `Payment_Method` for reference.